mne/viz/tests/test_raw.py

```diff
@@ -1214,3 +1219,35 @@ def test_plotting_memory_garbage_collection(raw, pg_backend):
 
     assert len(mne_qt_browser._browser_instances) == 0
     _assert_no_instances(MNEQtBrowser, "after closing")
+
+
+def test_plotting_scalebars(browser_backend, qtbot):
+    """Test that raw scalebars are not overplotted."""
+    ismpl = browser_backend.name == "matplotlib"
+    raw = mne.io.read_raw_fif(raw_fname).crop(0, 1).load_data()
+    fig = raw.plot(butterfly=True)
+    if ismpl:
+        ch_types = [text.get_text() for text in fig.mne.ax_main.get_yticklabels()]
+        assert ch_types == ["mag", "grad", "eeg", "eog", "stim"]
+        delta = 0.25
+        offset = 0
+    else:
+        qtbot.wait_exposed(fig)
+        for _ in range(10):
+            ch_types = list(fig.mne.channel_axis.ch_texts)  # keys
+            if len(ch_types) > 0:
+                break
+            qtbot.wait(100)  # pragma: no cover
+        # the grad/mag difference here is intentional in _pg_figure.py
+        assert ch_types == ["grad", "mag", "eeg", "eog", "stim"]
+        delta = 0.5  # TODO: Probably should also be 0.25?
+        offset = 1
```

Comment:
just curious why you skip checking "grad" for the pyqtgraph browser

Reply:
I don't think I do... `"grad"` is in the list and we `.pop(-1)` i.e. the `"stim"` channel. The others (grad, mag, eeg, eog) all get checked

Reply:
Oops you're right. I was thinking the nonzero second arg to enumerate was making it skip the first element (which of course isn't right)